readium/shared/src/main/java/org/readium/r2/shared/util/CursorList.kt

```diff
@@ -4,44 +4,51 @@ import org.readium.r2.shared.InternalReadiumApi
 
 /**
  * A [List] with a mutable cursor index.
+ *
+ * [next] and [previous] refer to the last element returned by a previous call
+ * to any of both methods.
+ *
+ * @param list the content of the [CursorList]
+ * @param index the index of the element that will initially be considered
```

Comment:
That _sounds_ good but that would break the specification we agreed on earlier:
>  [next] and [previous] refer to the last element returned by a previous call
    to any of both methods.
I'm not willing to open this can of worms once more.